synfig-studio/src/synfigapp/actions/waypointsetsmart.cpp

```diff
@@ -83,8 +83,13 @@ Action::WaypointSetSmart::get_param_vocab()
 {
 	ParamVocab ret(Action::CanvasSpecific::get_param_vocab());
 
+	ret.push_back(ParamDesc("value_desc",Param::TYPE_VALUEDESC)
+		.set_local_name(_("ValueDesc"))
+	);
```

Comment:
If it didn't exist before, it should be optional, right?

Reply:
oh yes, definetly. done :)